src/Falco/ConfigBuilder.fs

```diff
@@ -101,6 +107,11 @@ type ConfigBuilder (args : string[]) =
     member _.AddOptionalJsonFile (conf : ConfigBuilderSpec, filePath : string) =
         { conf with OptionalFiles = (JsonFile filePath) :: conf.OptionalFiles }
 
+    ///Adds optional user secrets to the ConfigurationBuilder.
+    [<CustomOperation("optional_user_secrets")>]
```

Comment:
Yes because it is generally only used in the Development environment. In the default web builder of the c# template where it sets up all the config for you it only adds optional user secrets if the env is development.